english/LifePuzzle

```diff
@@ -0,0 +1 @@
+Life is full of small puzzles and we try to find a way for it to fit to make sense of it all.
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.